web/src/unread_ops.js

```diff
@@ -168,7 +167,6 @@ function process_newly_read_message(message, options) {
     }
     notifications.close_notification(message);
     recent_view_ui.update_topic_unread_count(message);
-    inbox_ui.update();
```

Comment:
Looks like this one was previously sometimes called in a loop via `notify_server_messages_read` so this change is nice.